### ISSUE TITLE
Conflicting namespace/package name with upstream pymatgen

### ISSUE THREAD

**Issue by ml-evs** [Matgenix/pymatgen-io-lammps#1]
Just ran into an issue we should have perhaps foreseen... the current package clashes with the existing `pymatgen.io.lammps` namespace 🙃

**Comment by gbrunin** [comment]
Yes indeed, I discussed that with David yesterday. The last "real" (others than linting etc) developments in the original `pymatgen.io.lammps` are more than 1 year old, so the idea is to open an issue on pymatgen asking if it's ok to move all of that in our add-on (which is what I did already) and simply remove the existing namespace.

**Comment by JaGeo** [comment]
Yes, just do this and see what they say. I think you probably need to tag the previous developers and see if they are okay with it.
